deps/config/manifests/nodes.yml.template

```diff
@@ -1,17 +1,15 @@
-- kind: Node
-  name: quorum-node
+- kind: node
   specs:
-    rpc:
-      addr: http://quorum1:8545
-    tessera:
-      addr: http://tessera1:9080
-- kind: Node
-  name: besu-node
+    name: quorum-node
+    rpc: http://quorum1:8545
+    tessera: http://tessera1:9080
+
+- kind: node
   specs:
-    rpc:
-      addr: http://validator1:8545
-- kind: Node
-  name: geth-node
+    name: besu-node
+    rpc: http://validator1:8545
+
+- kind: node
   specs:
-    rpc:
-      addr: http://geth:8545
```

Comment:
This is a breaking change, we should deliverate this more before. DId you consider other RPC alternatives to be available such IPC?